projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.scss

```diff
@@ -7,7 +7,13 @@
   padding-left: 1px;
   padding-right: 1px;
 
-  .record {
+  .button {
+    margin-top: 8px;
+    margin-left: 88px;
```

Comment:
Strange margin.

Reply:
Had to put this so that it is aligned with the card boundary